soh/soh/SaveManager.cpp

```diff
@@ -614,11 +613,17 @@ void SaveManager::InitFileNormal() {
     gSaveContext.magicLevel = 0;
     gSaveContext.infTable[29] = 1;
     gSaveContext.sceneFlags[5].swch = 0x40000000;
+
+    // SoH specific
+    gSaveContext.backupFW = gSaveContext.fw;
     gSaveContext.pendingSale = ITEM_NONE;
     gSaveContext.pendingSaleMod = MOD_NONE;
 
+    // Boss Rush is set ahead of time in z_file_choose, otherwise init the save with the normal quest
     if (IS_BOSS_RUSH) {
         BossRush_InitSave();
+    } else {
+        gSaveContext.questId = QUEST_NORMAL;
```

Comment:
This is the actual fix (init-ing with quest normal)

Notice this has to be done after checking for boss rush. Ideally the boss rush stuff should be moved out/handled separately to this (similar to init for a rando save).